custom_components/waste_collection_schedule/waste_collection_schedule/source/samiljo_se.py

```diff
@@ -113,9 +113,14 @@ def fetch(self):
         streetcitylist = (self.street.lower(), self.city.lower())
         streetcityjoined = "|".join(streetcitylist)
         adresslistalines = adresslist.text.lower().splitlines()
+        A = None
         for line in adresslistalines:
             if streetcityjoined in line:
                 A = line.split("|")[3]
+                break
+        
+        if A is None:
+            return []
```

Comment:
Do not return an empty list but throw an Exception as this will show up in the logs while `return []` will just fail silently